schemas/ingest-metrics.v1.schema.json

```diff
@@ -40,7 +40,79 @@
             "type": "string"
           }
         },
-        "value": {},
+        "value": {
```

Comment:
Is there a reason why we decided to re-fill the `value` field here?

Reply:
Because type checking gets really annoying, also I realized that disabling that validation won't do us much good in terms of performance because large arrays will be sent as b64 strings anyways